internal/commands/commands.go

```diff
@@ -81,3 +81,19 @@ func getMirrors(config config.Config) map[string][]string {
 	}
 	return mirrors
 }
+
+func isTrustedBuilder(cfg config.Config, builder string) bool {
```

Comment:
This pattern is followed a number of times in the `commands`, and should be refactored out in all cases. If that shouldn't be part of this refactor, I'm happy to undo this change and leave it for another PR.